src/ansible_creator/resources/common/patterns/sample_pattern/meta/pattern.json

```diff
@@ -0,0 +1,44 @@
+{
+  "schema_version": "1.0",
+  "name": "weather_forecasting",
+  "title": "Weather Forecasting",
+  "description": "This pattern is designed to help get the weather forecast for a given location. It creates a project, EE, and job templates in automation controller to get the weather forecast.",
+  "short_description": "This pattern is designed to help get the weather forecast for a given location.",
+  "tags": ["weather", "forecasting"],
+  "aap_resources": {
+    "controller_project": {
+      "name": "Weather Forecasting",
+      "description": "Project for the Weather Forecasting pattern"
+    },
+    "controller_execution_environment": {
+      "name": "Weather Forecasting",
+      "description": "EE for the Weather Forecasting pattern",
+      "image_name": "weather-demo-ee",
+      "pull": "missing"
+    },
+    "controller_labels": ["weather", "forecasting"],
+    "controller_job_templates": [
+      {
+        "name": "Get Weather Forecast",
+        "description": "This job template gets the weather forecast for a given location.",
```

Comment:
Can this be rephrased to "gets the weather at the location of a provided airport code"